src/elements/content-explorer/ContentExplorer.js

```diff
@@ -356,15 +363,11 @@ class ContentExplorer extends Component<Props, State> {
             currentCollection: this.currentUnloadedCollection(),
             view: VIEW_METADATA,
         });
-
-        // Fetch the Metadata Query Results
-        this.api.getMetadataQueryAPI().queryMetadata(
+        this.metadataQueryAPIHelper = new MetadataQueryAPIHelper(this.api);
```

Comment:
If we're going to re-instantiate this class each time, does it need to be stored as an instance property of the parent class?

Reply:
The same instance will be used to update metadata.